doc/concept/overview.md

```diff
@@ -1,43 +1,89 @@
+The procedure for trading GOs is as follows:
 
-1. The energy produced is certified by a third party issuer compliant with a central authority (AIB), and the certificate is issued to the producer
-2. The certificate is then sold to the supplier
-3. The supplier then sells the certificate to the consumer
-4. The consumer can then use the certificate to claim/retire/cancel that the energy supplied is renewable
+1. A producer of energy requests a certificate for each MWh of energy produced, and a third party issuer, compliant with the central authority (AIB), issues the GO(s) to the producer
+2. The GO(s) is/are sold to a supplier
+3. The supplier sells the GO(s) to a consumer
+4. The consumer can then "use" the GO, by claiming, or [cancelling](https://en.energinet.dk/Energy-data/Guarantees-of-origin-el-gas-hydrogen/#accordion-cancellation), effectively attributing consumption of the renewable energy amount specified by the GO to the consumer.
 
-The existing GO system is based on the following principles:
+To summarize, the existing GO system is based on the following principles:
 - Yearly certification
 - Volume based accounting
 - Exchange of certificates between parties through a centralised system AIB-hub
-- The certificate is a tradable asset
-- No direct physical link between the certificate and the energy supplied are required
-
-With the penetration of renewable energy increasing, at times more than 100% of certain grid areas are supplied by renewables, it brings a number of challenges to the existing system: 
-- Makes it hard to balance the energy grid to 50Hz
-- Creates bottlenecks in the existing grid 
-- Requires loadbalancing that is less centralized 
+- The certificate is a tradeable asset
+- No direct physical link between the certificate and the energy supplied is required
+#### Challenges  
+At times, certain grid areas experience a supply by renewables that is [bigger than the grid capacity](https://www.caiso.com/documents/curtailmentfastfacts.pdf), which brings a number of [challenges to the existing, physical grid](https://www.rff.org/publications/explainers/renewables-101-integrating-renewables/): 
+- It becomes [difficult to stabilize the grid frequency](https://www.engineering.com/story/grid-frequency-stability-and-renewable-power) in the electricity grid.
+- Insufficient capacity of the existing grid creates [bottlenecks](https://www.zerohedge.com/energy/grid-bottlenecks-could-derail-europes-renewable-energy-boom), limiting transportation between certain areas 
+- [Less centralized load balancing](https://research.rug.nl/en/publications/local-balancing-of-the-electricity-grid-in-a-renewable-municipali) might be necessary 
 - Makes it hard to validate the actual energy supplied to power-to-x applications and the origin of the losses in the process
+#### Drawbacks
```

Comment:
General comment to this section. We have to be careful not to criticize too much, even though the evidence and argumentation you are bringing in here calls for action...lets see if we can make it a bit more smooth

Reply:
Thanks for the feedback @NicolasENDK - I will prioritize making this more smooth. I realize that this section is also quite lengthy, which seems unnecessary given that this doc is an overview of "the what and why" for project-origin